sdks/python/apache_beam/io/gcp/tests/bigquery_matcher.py

```diff
@@ -46,9 +47,11 @@
 MAX_RETRIES = 5
 
 
-def retry_on_http_and_value_error(exception):
+def retry_on_http_timeout_and_value_error(exception):
   """Filter allowing retries on Bigquery errors and value error."""
-  return isinstance(exception, (GoogleCloudError, ValueError))
+  return isinstance(exception, (GoogleCloudError,
+                                ValueError,
+                                concurrent.futures.TimeoutError))
```

Comment:
Where is this `TimeoutError` being raised? I didn't see it in the failed tests' logs .